documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -221,18 +221,21 @@ The final version of the evaluation report may only be submitted to the Applican
 8. The Applicant may only make the evaluation report available to third parties in full text, stating the date of issue, and pass it on after written approval by the Evaluation Facility.
 
 ### 3.6 Change Certification     
-1. Change Certifications can be carried out at the Applicant\'s request.
-2. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
-3. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
+1. After a component has been certified, the applicant must inform the Certification Body and the Evaluation Facility (in the case of Assurance Level 2) of any changes made to the component. The applicant must report the changes and fill in the impact assessment provided by the Certification Body.
+2. Change Certifications can also be carried out at the Applicant\'s request.
+3. During a change certification, the Certification Body must determine whether the certified operational environment or component continues to meet the IDS requirements of the originally certified Trust Level after changes have been made.
+4. For small changes, e.g. bug fixes or security patches for components, the change can be certified with minimal effort in form of a maintenance certification.
```

Comment:
A maintenance certification does not extend the validity of the old certificate. This is not yet mentioned, since the statements below concerning the validity only aplly to re-certification and renewal certification.

Reply:
I re-read the text and if items 7 - 9 are in regards to re-certification **and** maintenance, no changes are necessary. This was just not clear to me, when reading the text the first time.